docs/content/en/docs/operator-manual/piped/installation/installing-on-fargate.md

```diff
@@ -0,0 +1,149 @@
+
+## Installation
+
+- Preparing a piped configuration file as the following:
+
+``` yaml
+apiVersion: pipecd.dev/v1beta1
+kind: Piped
+spec:
+  projectID: {PROJECT_ID}
+  pipedID: {PIPED_ID}
+  pipedKeyData: {BASE64_ENCODED_PIPED_KEY}
+  webAddress: {CONTROL_PLANE_WEB_ADDRESS}
+  # Write in a format like "host:443" because the communication is done via gRPC.
+  apiAddress: {CONTROL_PLANE_API_ADDRESS}
+
+  git:
+    sshKeyData: {BASE64_ENCODED_PRIVATE_SSH_KEY}
+
+  repositories:
+    - repoId: {REPO_ID_OR_NAME}
+      remote: git@github.com:{GIT_ORG}/{GIT_REPO}.git
+      branch: {GIT_BRANCH}
+
+  # Optional
+  # Enable this Piped to handle ECS application.
+  cloudProviders:
+    - name: ecs-dev
+      type: ECS
+      config:
+        region: {ECS_RUNNING_REGION}
+
+  # Optional
+  # Uncomment this if you want to enable this Piped to handle Terraform application.
+  #  - name: terraform-dev
+  #    type: TERRAFORM
+
+  # Optional
+  # Uncomment this if you want to enable SecretManagement feature.
+  # https://pipecd.dev/docs/user-guide/secret-management
+  # secretManagement:
+  #   type: KEY_PAIR
+  #   config:
+  #     privateKeyData: {BASE64_ENCODED_PRIVATE_KEY}
+  #     publicKeyData: {BASE64_ENCODED_PUBLIC_KEY}
+```
+
+- Create a new secret in [SecretManager](https://aws.amazon.com/secrets-manager/) to store above configuration data securely
+
+```console
+aws secretsmanager create-secret --name PipedConfig \
+  --description "Configuration of piped running as ECS Fargate task" \
+  --secret-string file://piped-config.yaml
+```
+
+Note: Make sure your account has permission to execute CreateSecret operation or you may get `AccessDeniedException` error from AWS Api. In that case, you can try creating secret via AWS Web console.
+
+- Prepare task definition for your piped task. Basically, you can just define your piped TaskDefinition as normal TaskDefinition, the only thing that needs to be beware is, to enable your piped accesses it's configuration we created as a secret on above, you need to add `secretsmanager:GetSecretValue` policy to your piped task `executionRole`. Read more in [Required IAM permissions for Amazon ECS secrets](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/specifying-sensitive-data-secrets.html).
+
+A sample TaskDefinition for Piped as following.
+
+```json
+{
+  "family": "piped",
+  "executionRoleArn": "{PIPED_TASK_EXECUTION_ROLE_ARN}",
+  "containerDefinitions": [
+    {
+      "name": "piped",
+      "image": "gcr.io/pipecd/piped:{{< blocks/latest_version >}}",
+      "command": [
+        "echo $configData > /etc/piped/config.yaml; piped --config-file=/etc/piped/config.yaml --metrics=true"
```

Comment:
Metrics is already enabled by default so I guess we can omit its arg in this command.

Reply:
```suggestion
        "echo $configData > /etc/piped/config.yaml; piped --config-file=/etc/piped/config.yaml"
```